cmd/nrt/config_split/config.toml

```diff
@@ -5,11 +5,13 @@ TrimColAfterSplit = true # (bool), if true, designated Trim Columns apply to Spl
 WalkSubFolders = true # (bool), if true, splitter process every file including the file in sub-folders; otherwise, ignores sub-folder files.
 
 [Trim]
-Columns = ["School", "YrLevel", "Domain"] # (string array), which columns to be removed from original csv file.      
+Columns = ["School", "YrLevel", "Domain"] # (string array), which columns to be removed from original csv file.                    
 Enabled = true # (bool), turn on/off Trim function.
-OutFolder = "./out/" # (string), in which folder trimmed csv files should be output.
+OutFolder = "./out_trim/" # (string), in which folder trimmed csv files should be output.
```

Comment:
No, the folder should be named out, for consistency with existing program.  The fact that the out folder is a result of trimming should be hidden from users. Is there a compelling reason to rename it?